js/terms.json

```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Date received",
+    "definition": "The date a committee (or a person acting on the committee’s behalf) takes possession of the contribution. 11 CFR 102.8(a). This date is used for FEC reporting."
+  },
+  {
+    "term": "Delegate",
+    "definition": "An individual who is or seeks to become a delegate to a national nominating convention or to a state, district or local convention, caucus or primary held to select delegates to a national nominating convention. 11 CFR 110.14(b)(1)."
+  },
+  {
+    "term": "Delegate committee",
+    "definition": "A group organized for the purpose of influencing the selection of one or more delegates. The term includes a group of delegates, a group of individuals seeking to become delegates and a group of individuals supporting delegates. 11 CFR 110.14(b)(2)."
+  },
+  {
+    "term": "Direct",
+    "definition": "For purposes of 11 CFR Part 300, to direct means to guide, directly or indirectly, a person who has expressed an intent to make a contribution, donation, transfer of funds or otherwise provide anything of value, by identifying a candidate, political committee or organization for the receipt of such funds or things of value. The contribution, donation, transfer or thing of value may be provided directly or through an intermediary. Direction does not include merely providing information or guidance as to the applicability of a particular law or regulation. 11 CFR 300.2(n)."
+  },
+  {
+    "term": "Direct mail",
+    "definition": "Any mailings made by a commercial vendor or made from a commercial list. 11 CFR 100.87(a), 100.89(a) 100.147(a) and 100.149(a)."
+  },
+  {
+    "term": "Earmarked contribution",
+    "definition": "A contribution that the contributor directs (either orally or in writing) to or on behalf of a clearly identified candidate or authorized committee through an intermediary or conduit. Earmarking may take the form of a designation, instruction or encumbrance, and it may be direct or indirect, express or implied. 11 CFR 110.6."
+  },
+  {
+    "term": "Election",
+    "definition": "Any one of several processes by which an individual seeks nomination for election, or election, to federal office. They include: a primary election, including a caucus or convention that has authority to select a nominee; a general election; a runoff election; and a special election held to fill a vacant seat. 11 CFR 100.2."
+  },
+  {
+    "term": "Election cycle",
+    "definition": "The period beginning the day after the previous general election for a given federal office and ending on the date of the general election for that office. The number of years in an election cycle differs according to the federal office sought. The election cycle spans two years for House candidates; four years for presidential candidates; and six years for Senate candidates. See 11 CFR 100.3(b)."
```

Comment:
I would leave the citations as is. The "see" is usually going to go with regulations that are a little open to interpretation, where the ones without it are more slam dunk. So if we are using "see", chances are it was recommended by our lawyers.